Pod/Classes/Banner.swift

```diff
@@ -114,18 +128,32 @@ open class Banner: UIView {
     }
     
     /// The label that displays the banner's title.
-    @objc open let titleLabel: UILabel = {
+    @objc public let titleLabel: UILabel = {
         let label = UILabel()
-        label.font = UIFont.preferredFont(forTextStyle: UIFontTextStyle.headline)
+      
+        #if swift(>=4.2)
+        let textStyle = UIFont.TextStyle.headline
```

Comment:
This one can just be changed to ` UIFont.preferredFont(forTextStyle: .headline)` and be backwards-compatible.

Reply:
True